sympy/concrete/summations.py

```diff
@@ -670,6 +673,95 @@ def is_absolutely_convergent(self):
         Sum.is_convergent()
         """
         return Sum(abs(self.function), self.limits).is_convergent()
+    @staticmethod
+    def is_expr_convergent(expr):
+        """
+        Checks for convergence of an expression.
+        Parameters
+        ----------
+        expr : TYPE
+            DESCRIPTION.
+
+        Raises
+        ------
+        TypeError
+            DESCRIPTION.
+
+        Returns
+        -------
+        bool
+            DESCRIPTION.
+
+        """
+        if isinstance(expr, (Mul,Add,Pow,Zero,Infinity)):
+            args = expr.args
+            a,b,c=0,0,0
+            if isinstance(expr ,Add):
+                for i in range(len(args)):
+                    if(isinstance(args[i],Sum)):
```

Comment:
No need for brackets around if in python
For commas, the followed convention is no space before the comma and a single space after it